guides/source/active_record_validations.md

```diff
@@ -188,13 +193,13 @@ false` as an argument. This technique should be used with caution.
 [`upsert`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert
 [`upsert_all`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert_all
 
-### `valid?` and `invalid?`
+### Checking Validity
 
-Before saving an Active Record object, Rails runs your validations.
-If these validations produce any errors, Rails does not save the object.
+Before saving an Active Record object, Rails runs your validations, and
+if these validations produce any errors, then Rails will not save the object.
```

Comment:
If was going to suggest using another word than `errors` as this might confuse novices with programming errors.
However, `errors` are heavily used in the validations implementation, so that ship has sailed.
Not sure if it makes sense use "validation errors" instead (maybe just for the first occurences)?
```suggestion
if these validations produce any validation errors, then Rails will not save the object.
```